sql/core/src/test/resources/test-data/unescaped-quotes-unescaped-delimiter.csv

```diff
@@ -0,0 +1,3 @@
+c1,c2
+"a,""b,c","xyz"
+"a,b,c","x""yz"
```

Comment:
Since we're here, can we make the test self-contained instead of relying on the external file? I generally prefer to make the test self-contained to make it easier to read.

Reply:
Address ca79a48 try to fix this.

Reply:
Address ca79a48 try to fix this